common/common.cpp

```diff
@@ -559,6 +563,16 @@ bool gpt_params_parse_ex(int argc, char ** argv, gpt_params & params) {
 #else
             fprintf(stderr, "warning: llama.cpp was compiled without cuBLAS. It is not possible to set a tensor split.\n");
 #endif // GGML_USE_CUBLAS
+        } else if (arg == "--vram-budget") {
```

Comment:
vram-budget is a parameter. Interesting. Trying to optimize for GPU memory I see.